postreise/analyze/demand.py

```diff
@@ -0,0 +1,44 @@
+from powersimdata.network.usa_tamu.usa_tamu_model import area_to_loadzone
+
+from postreise.analyze.generation.summarize import (
+    get_generation_time_series_by_resources,
+)
+
+
+def get_demand_time_series(scenario, area, area_type=None):
+    """Get time series demand in certain area of a scenario
+
+    :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
+    :param str area: one of: *loadzone*, *state*, *state abbreviation*,
+        *interconnect*, *'all'*
+    :param str area_type: one of: *'loadzone'*, *'state'*,
+        *'state_abbr'*, *'interconnect'*
```

Comment:
Should we simplify this docstring (and make it less likely to drift) by simply saying that these inputs must conform to `area_to_loadzone`? In case we do something like add 'timezone' as another selection criteria to `area_to_loadzone`.

Reply:
The reason I did copy and paste doc strings from `area_to_loadzone` to here instead of simply referring to `area_to_loadzone` is this is a function that a user may call directly and one could simply do `print(get_demand_time_series.__doc__)` to see what parameters are expecting from the function.

Reply:
That's fair. Being explicit is fine, let's just remember this if/when we ever change `area_to_loadzone`!

Reply:
😅